test/functional/test_framework/test_framework.py

```diff
@@ -139,15 +144,18 @@ def main(self):
             self.options.tmpdir = tempfile.mkdtemp(prefix="test")
         self._start_logging()
 
-        success = False
+        success = TestStatus.FAILED
 
         try:
             self.setup_chain()
             self.setup_network()
             self.run_test()
-            success = True
+            success = TestStatus.PASSED
         except JSONRPCException as e:
             self.log.exception("JSONRPC error")
+        except SkipTest as e:
+            self.log.warning("Test Skipped: %s" % e.message)
```

Comment:
Yes - this log is replacing the logs in the individual test cases (rpcbind_test.py and zmq_test.py), which are currently warnings.

The reason is that we always want the user to know if a test has been skipped.